src/gc/unix/gcenv.unix.cpp

```diff
@@ -517,6 +523,14 @@ void GCToOSInterface::GetMemoryStatus(uint32_t* memory_load, uint64_t* available
         *available_page_file = 0;
 }
 
+// Get size of an OS memory page
+void GCToOSInterface::GetPageSize()
+{
+    static pageSize = (sysconf( _SC_PAGE_SIZE ) != -1) ? sysconf( _SC_PAGE_SIZE ) : 4096;
```

Comment:
I believe function local static constructors are a problematic construct. They are not always thread safe, have varying perf characteristics, ... .

I think this should be rather initialize in `GCToOSInterface::Inititialize`.